src/ucs/sys/compiler_def.h

```diff
@@ -126,6 +126,12 @@
         ucs_container_of(_ptr, _type, super); \
     })
 
+/**
+ * @return Size of _member in _type. _type is a structure type.
+ */
+#define ucs_sizeof(_type, _member) \
```

Comment:
maybe ucs_field_sizeof ?

Reply:
ok, thought about `ucs_member_sizeof()` name :).  Let's go with `ucs_filed_sizeof`

Reply:
either one is good